src/components/TasksTable/index.jsx

```diff
@@ -92,33 +104,54 @@ export default class TasksTable extends Component {
     });
   };
 
+  handleChipClick = tag => event => {
+    const query = this.getQuery();
+    const newQuery =
+      query.tag === tag ? omit(['tag'], query) : { ...query, tag };
+
+    this.props.history.push({
+      search: `?${stringify(newQuery)}`,
+    });
+
+    event.preventDefault();
+  };
+
   render() {
     const { items, classes } = this.props;
-    const { sortBy, sortDirection } = this.getQuery();
-    const data = this.getTableData(sortBy, sortDirection, items);
+    const iconSize = 14;
+    const query = this.getQuery();
+    const { sortBy, sortDirection, tag } = query;
+    const data = this.getTableData(sortBy, sortDirection, tag, items);
 
     return (
       <div className={classes.tableWrapper}>
         <DataTable
           title="Bugs & Issues"
           items={data}
           renderRow={item => (
-            <TableRow
-              hover
-              target="_blank"
-              rel="noopener noreferrer"
-              tabIndex={-1}
-              key={item.summary}
-              component="a"
-              href={item.url}
-              className={classes.link}>
+            <TableRow hover tabIndex={-1} key={item.summary}>
               <TableCell component="th" scope="row">
                 {item.project}
               </TableCell>
-              <TableCell className={classes.summary}>{item.summary}</TableCell>
+              <TableCell
+                className={classes.summary}
+                target="_blank"
+                rel="noopener noreferrer"
+                component="a"
+                href={item.url}>
+                {item.summary}
+                <LinkIcon size={iconSize} className={classes.icon} />
```

Comment:
Did you look at tc-web on how we handle table cells which have links?

_Screenshots from the Worker Types view (http://localhost:5080/provisioners/aws-provisioner-v1)_

![screen shot 2018-06-29 at 8 59 26 am](https://user-images.githubusercontent.com/3766511/42093577-d72ca58a-7b7a-11e8-86d3-e89070e0fa2b.png)

![screen shot 2018-06-29 at 8 59 56 am](https://user-images.githubusercontent.com/3766511/42093591-dbf569da-7b7a-11e8-977f-8f9166b7afe5.png)